packages/analytics-js/src/components/utilities/loadOptions.ts

```diff
@@ -130,27 +130,34 @@ const getSourceConfigURL = (
   let searchParams = defSearchParams;
   let pathname = '/sourceConfig/';
   let hash = '';
-  try {
-    const configUrlInstance = new URL(configUrl);
-    if (!(removeTrailingSlashes(configUrlInstance.pathname) as string).endsWith('/sourceConfig')) {
-      configUrlInstance.pathname = `${
-        removeTrailingSlashes(configUrlInstance.pathname) as string
-      }/sourceConfig/`;
-    }
-    configUrlInstance.pathname = removeDuplicateSlashes(configUrlInstance.pathname);
-
-    defSearchParams.forEach((value, key) => {
-      if (configUrlInstance.searchParams.get(key) === null) {
-        configUrlInstance.searchParams.set(key, value);
+  // Ideally, this check is not required but URL polyfill
+  // doesn't seem to throw errors for empty URLs
+  // TODO: Need to improve this check to find out if the URL is valid or not
```

Comment:
Do we realy have too? The customers will not only use IE11. If the url is worng it will throw on other borwsers for them. I would not raise the filesize only for this one if there is ot eazy fix

Reply:
It's also needed for other use cases as well. So, we have to explore some really light weight solutions.